cmd/prettylog/README.md

```diff
@@ -0,0 +1,86 @@
+# Zerolog PrettyLog
+
+This is a basic CLI utility that will colorize and pretty print your JSON logs.
+
+## Problem
+
+When using zerolog it's sometimes difficult to do an environment based log formatting.
+For production purposes it is useful to have JSON logs to pipe them through Kibana for efficient log entry analysis.
+
+However, when using such applications in development environment there is little use for structured JSON logs.
+Humans are visual beings so the preferred format should be ordinary `Console` format with colors if possible.
+
+Zerolog ships with `ConsoleWriter` which does exactly that. But modifying your code to get pretty printed and colorized
+logs is somewhat difficult.
```

Comment:
A simpler solution to the problem described here is to detect if the stdout is a terminal and automatically enable console mode if it is. This is a trick I may add as conditional writer in the lib.

I still think this tool is useful when you need to tail zerolog produced output in prod for instance, or in settings when you need both JSON and pretty output.

Could you change the text of the readme to remove the part about why you need this tool so it does not come as the recommended solution to solve this problem?

Reply:
Yes - of course. I updated the README.md - now it's just a description and usage.

Sorry for taking so long - was on vacation during the summer :)